expr/span.go

```diff
@@ -0,0 +1,86 @@
+// Copyright observIQ, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package expr
+
+import (
+	"go.opentelemetry.io/collector/pdata/ptrace"
+)
+
+// Span specific fields for use in expressions
+const (
+	SpanKindField          = "trace_kind"
+	SpanStatusCodeField    = "trace_status_code"
+	SpanStatusMessageField = "trace_status_message"
+	SpanDurationField      = "span_duration_ms"
```

Comment:
Do these not cause linter errors? I would expect so since they're exported.

Reply:
One comment on the group is satisfies the linter here. I think the one comment up there is sufficient enough, eventually when the actual processors are implemented, these will be documented there.